### ISSUE TITLE
Support shell autocompletions

### ISSUE THREAD

**Issue by Riey** [Riey/cargo-feature#18]
Like bash, zsh

**Comment by kjamrozy** [comment]
I would like to implement this feature. However AFAIK it's not possible to install shell completions automatically(because there is no post-install hook in cargo to execute a custom script). So there must be some command to install them on demand. Maybe something like this `cargo feature --install-shell-completions` . What do you think?

**Comment by Riey** [comment]
> I would like to implement this feature. However AFAIK it's not possible to install shell completions automatically(because there is no post-install hook in cargo to execute a custom script). So there must be some command to install them on demand. Maybe something like this `cargo feature --install-shell-completions` . What do you think?

I agree. Maybe package manager do that automatically for nix and arch users.
